build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:7.0.0"
+        classpath 'com.android.tools.build:gradle:7.0.1'
```

Comment:
Can extract this in a separate PR. Changing build conflig files has nothing to do with displaying files on UI. It is a good practice no to mix changes that does different things in the same PR.